hapi-utils/src/main/java/com/hedera/services/legacy/proto/utils/SignatureVerifier.java

```diff
@@ -238,9 +239,9 @@ public static void main(String[] args)  {
     byte[] encKey = null;
     try (FileInputStream  keyfis = new FileInputStream(args[0])) {
         encKey = new byte[keyfis.available()];
-        keyfis.read(encKey);
+        int bytesRead = keyfis.read(encKey);
```

Comment:
Adding `bytesRead` here will make it unused. As for the `read` function, it is always safer to check the `bytesRead` with the total number of bytes that we intended to read, which is `encKey.length`

Reply:
Added a warn when they mismatch.